wrappers/wrapper.rb

```diff
@@ -572,6 +573,66 @@ def detect_unfeasible_services(vrp)
       unfeasible
     end
 
+    def relation_with_no_reachable_vehicle(vrp, relation)
+      return false unless Models::Relation::POSITION_TYPES.include?(relation.type)
+
+      vrp.vehicles.each{ |v|
+        next if reachable_service_sequence?(vrp, v, relation.linked_services)
+
+        relation.linked_services.each{ |service|
+          service.vehicle_compatibility[v.id] = false
+        }
```

Comment:
In theory/practice, we can plan the beginning of an order/sequence relation and skip the tail of it. Here we completely remove that possibility? Is it voluntary?